### PR TITLE
fix: add release name as prefix to chart resource names during deployment

### LINKED ISSUE #388: Charts[GKE-BUG] Allow the chart to be deployed with different release-name
Currently it assumes that the helm chart is deployed with release name `fst`. However, we should be able to override this with env variable RELEASE_NAME. For example,

```make deploy-network RELEASE_NAME=fst2```

### PR DESCRIPTION
**Pull request by leninmehedy**
## Description

This pull request changes the following:

- allow custom release name as prefix to all resources same as mirror-node resources
- update makefile scripts to use the RELEASE_NAME env var from env or .env file

Below screenshot shows an example deployment with `RELEASE_NAME=fst`:

Screenshot 2023-10-09 at 9 30 40 AM

### Limitations:
I started working on this with the anticipation that this could allow deploying FST chart multiple times in the same namespace. For example, if a user has access to a single namespace, he/she could deploy the FST chart multiple times in the same namespace for different test scenario and avoid the explosion namespaces in the shared cluster.

However, it appears dependent charts e.g. mirror-node creates some secrets without release name prefix (e.g. it creates "mirror-passwords" instead of "{{.Release.Name}}-mirror-passwords)", so it blocks us from deploying the chart multiple times in the same namespace)

So at the end this PR doesn't add much value except the fact that it is now consistent with mirror-node resource naming. However, if mirror-node updates their secrets with release name prefix, we would be able to deploy the same chart multiple times for different test scenario and avoid the "namespace explosion" in the cluster...

Screenshot 2023-10-09 at 9 36 19 AM

### Related Issues

- Closes #388

### CODE REVIEW

**Comment by leninmehedy** [comment]
After discussing with Nathan we decided not to proceed in this direction because of the following reasons:
- Third-party dependent chart resource may not support multiple deployment in the same namespace (e.g. we already discovered issue with the secrects created by mirror-node)
- Namespace is the best and standard way to segregate test environments. So all our tests environments will be segregated using namespace only to avoid confusion.

So in summary, we shall assume that the chart can be deployed only once in a namespace. 

However, all FST resources should be labeled with the test related metadata (see #409)
